templates/default-layout.lucius

```diff
@@ -60,7 +60,7 @@ body {
 
 .col ul,
 .col ol {
-  padding-left: 25px;
+  padding-left: 45px;
```

Comment:
Do you know what else is affected by this? I think this affects display of type class members and data constructors, and perhaps also module listings on package homepages, for example. Do they still look ok? Would it make sense to use a class to make this rule a little more specific?